app/views/search/index.html.erb

```diff
@@ -122,8 +122,11 @@
                 <%= heading_tag(level: 2, class: 'result__heading-text t-name', data: { results_module_heading: '' }) do %>
                   <span class="result__heading-span"><%= firm.name %></span>
                 <% end %>
-                <% if @form.face_to_face? %>
+
+                <% if @form.face_to_face? && firm.closest_adviser > 1 %>
```

Comment:
This should go into the `FirmResult` instead. That is acting as a decorator so would be perfect for that. We can sit down and work this out together.